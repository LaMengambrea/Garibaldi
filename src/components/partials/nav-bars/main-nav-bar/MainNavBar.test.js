import Navbar from './navbar';
import React from 'react';
import ReactDOM from 'react-dom';

test('Navbar renders', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Navbar />, div);
});