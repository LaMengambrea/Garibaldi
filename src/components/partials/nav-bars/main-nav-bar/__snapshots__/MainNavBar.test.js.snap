exports[`test MainNavBar renders 1`] = `
<div
  className="MainNavBar navbar navbar-default navbar-static-top">
  <div
    className="container">
    <div
      className="row">
      <div
        className="col-xs-12 col-md-12">
        <ul
          className="nav navbar-nav navbar-left">
          <li>
            <a
              onClick={[Function]}
              style={Object {}}>
              Archivo
            </a>
          </li>
          <li>
            <a
              onClick={[Function]}
              style={Object {}}>
              Artistas
            </a>
          </li>
          <li>
            <div
              className="SearchBar">
              <i
                className="glyphicon glyphicon-search searchElement"
                id="filterIcon" />
              <input
                className="searchElement"
                id="filterInput"
                placeholder="Filtrar información..."
                type="text" />
              <select
                className="searchElement"
                id="filterTypeSelect">
                <option
                  value="1">
                  Nombre de la pieza
                </option>
                <option
                  value="2">
                  Año
                </option>
                <option
                  value="3">
                  Artista
                </option>
              </select>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
